app/helpers/books_helper.rb

```diff
@@ -0,0 +1,4 @@
+# frozen_string_literal: true
+
+module BooksHelper
```

Comment:
be careful also with empty files, we use to delete them.

Reply:
removing this...